controller-server/src/main/java/com/yahoo/vespa/hosted/controller/versions/VersionStatus.java

```diff
@@ -263,8 +260,8 @@ private static VespaVersion createVersion(DeploymentStatistics statistics,
         }
 
         // Preserve existing commit details if we've previously computed status for this version
-        var commitSha = controllerVersion.commitSha();
-        var commitDate = controllerVersion.commitDate();
+        var commitSha = latestVersion.commitSha();
+        var commitDate = latestVersion.commitDate();
```

Comment:
Shouldn't this be the details of the version `statistics.version()`?

Reply:
Ideally yes, but the details only exist in a `ControllerVersion` and the first time a new `statistics.version()` appears it will be due to a controller upgrade...